pertpy/tools/_differential_gene_expression/_base.py

```diff
@@ -243,7 +299,7 @@ def _map_genes_categories_highlight(
             if varm_key is None:
                 raise ValueError("Please pass a .varm key to use for plotting")
 
-            raise NotImplementedError("Anndata not implemented yet")
+            raise NotImplementedError("Anndata not implemented yet")  # TODO: Implement this
```

Comment:
What would be the anndata usecase here actually? DE results stored in `.varm`? 
This is not supported by the DE tools either, so not sure this is required.

Reply:
I'm actually not sure myself - that part was already implemented (or rather not implemented in this case). @Zethson, what exactly did you have in mind here? Should I remove it?